Integrations/integration-nmap.yml

```diff
@@ -0,0 +1,98 @@
+description: Run nmap scans with the given parameters
+configuration: []
+script:
+  script: |
+    from libnmap.process import NmapProcess
+    from libnmap.parser import NmapParser
+    from libnmap.reportjson import ReportEncoder
+
+    if demisto.command() == 'test-module':
+        demisto.results('ok')
+        sys.exit(0)
+    if demisto.command() == 'nmap-scan':
+        nm = NmapProcess(argToList(demisto.args()['targets']), options=demisto.args()['options'])
+        rc = nm.run()
+        if rc != 0:
+            demisto.results({'Type': entryTypes['error'], 'ContentsFormat': formats['text'], 'Contents': 'Unable to execute - ' + nm.stderr})
+            sys.exit(0)
+        r = NmapParser.parse(nm.stdout)
+        md = '## ' + r.summary + '\n\n'
+        hosts = []
+        for host in r.hosts:
+            h = {}
+            if len(host.hostnames):
+                tmp_host = host.hostnames.pop()
+                h['Hostname'] = tmp_host
+            else:
+                tmp_host = host.address
+
+            h['Address'] = host.address
+            h['Status'] = host.status
+            svc = []
+            md += "- Nmap scan report for {0} ({1})".format(tmp_host, host.address) + '\n'
+            md += "- Host is {0}.".format(host.status) + '\n'
+            md += 'Port | State | service\n-|-|-\n'
+            for serv in host.services:
+                pserv = "{0} / {1} | {2} | {3}".format(
+                        str(serv.port),
+                        serv.protocol,
+                        serv.state,
+                        serv.service)
```

Comment:
service might contain chars that needs to be escaped, as they might be MD special chars

Reply:
Changed to use standard `tableToMarkdown`